### PR TITLE
[HIBIKE] Servo was improperly merged. Fixed and reintegrated

### PR DESCRIPTION
**Pull request by JHGZhou**

### CODE REVIEW

File: hibike/hibike_tester.py [hunk 1]
```diff
@@ -30,5 +30,13 @@ def disable(self):
 if __name__ == '__main__':
     h = Hibike()
     h.enumerate()
```
Comment (yizow) [inline hibike/hibike_tester.py]: Testing file changes?

File: hibike/devices/ServoControl/servo_control.cpp [hunk 2]
```diff
@@ -20,12 +21,6 @@ void enable(int num) {
   servos[num].attach(pins[num]);
 }
 
-void setup() {
-  hibike_setup(500); // Time in milliseconds before timeout on heartbeat
-  disableAll();
```
Comment (yizow) [inline hibike/devices/ServoControl/servo_control.cpp:25]: Are we removing disable all?

**Review by yizow — CHANGES_REQUESTED** [review]

File: hibike/hibike_tester.py (hunk 1 above)
Comment (JHGZhou) [reply hibike/hibike_tester.py]: Oops, reverted to original state

File: hibike/devices/ServoControl/servo_control.cpp (hunk 2 above)
Comment (JHGZhou) [reply hibike/devices/ServoControl/servo_control.cpp:25]: Added disable all back in

**Review by yizow — APPROVED** [review]

**Pull request merged**